src/main/java/module/transfer/XMLParser.java

```diff
@@ -44,14 +44,15 @@ public static List<PlayerTransfer> getAllPlayerTransfers(int playerId) {
 
     	final String xml = MyConnector.instance().getTransfersForPlayer(playerId);
```

Comment:
Well, the PR fixes the bug but I would like to know what the exact XML is that cannot be parsed.

Reply:
As far as i understood the issue this is a followup of an http request error - maybe because ht was down for a while. Then an empty xml string is returned from the MyConnector function.